theano/tensor/nnet/abstract_conv.py

```diff
@@ -109,7 +109,16 @@ def get_conv_shape_1axis(image_shape, kernel_shape,
         pad = border_mode
         if pad < 0:
             raise ValueError("border_mode must be >= 0")
-    out_shp = (image_shape + 2 * pad - kernel_shape) // subsample + 1
+
+            # In case of symbolic shape, we want to build the smallest graph
```

Comment:
I am able to understand what this code does, but how would it help in speeding up ?